commons/constants.py

```diff
@@ -49,6 +49,7 @@
 POD_NAME_PREFIX = "cortx-data"
 CORTX_DATA_NODE_PREFIX = "cortx-data-headless-svc-"
 SERVER_POD_NAME_PREFIX = "cortx-server"
+CORTX_CLIENT_SVC_POSTFIX = "cortx-client-headless.cortx.svc.cluster.local"
```

Comment:
Can we change it to more general similar to another name?

Reply:
Also please add test run log

Reply:
This is common to every motr client service name also we needed this info to find out the corresponding primary's node client name

Reply:
> Also please add test run log

Added logs.